deps/Makefile

```diff
@@ -1436,9 +1431,10 @@ UTF8PROC_SRC_TARGET = $(UTF8PROC_SRC_DIR)/libutf8proc.a
 UTF8PROC_OBJ_LIB    = $(build_libdir)/libutf8proc.a
 UTF8PROC_OBJ_HEADER = $(build_includedir)/utf8proc.h
 UTF8PROC_OBJ_TARGET = $(UTF8PROC_OBJ_LIB) $(UTF8PROC_OBJ_HEADER)
+UTF8PROC_CFLAGS     = "-O2 -std=c99 $(fPIC) -DUTF8PROC_EXPORTS"
```

Comment:
appveyor not so happy with the way the quoting turned out on this one. Man, makefiles shouldn't be written by humans.